app/services/articles/feeds/timeframe.rb

```diff
@@ -1,12 +1,14 @@
 module Articles
   module Feeds
     module Timeframe
-      def self.call(timeframe, tag: nil, number_of_articles: Article::DEFAULT_FEED_PAGINATION_WINDOW_SIZE, page: 1)
+      def self.call(timeframe, tag: nil, minimum_score: -20,
+                    number_of_articles: Article::DEFAULT_FEED_PAGINATION_WINDOW_SIZE, page: 1)
         articles = ::Articles::Feeds::Tag.call(tag)
 
         articles
           .where("published_at > ?", ::Timeframe.datetime(timeframe))
           .includes(:distinct_reaction_categories)
+          .where("score > ?", minimum_score)
```

Comment:
I decided to change the class behaviour, because it's only used or the affected feeds.